test/cql-pytest/test_tablets.py

```diff
@@ -126,3 +126,42 @@ def test_tablets_are_dropped_when_dropping_table(cql, test_keyspace, skip_withou
 
     cql.execute(f"DROP TABLE {test_keyspace}.{table_name}")
     verify_tablets_presence(cql, test_keyspace, table_name, expected=False)
+
+
+# Test that when a view of a tablets-enabled table is dropped, all of its tablets are dropped with it.
+@pytest.mark.parametrize("attempt_drop_table", [False, True])
```

Comment:
Nitpick (but again, I don't insist, it's not important):
You didn't need to make attempt_drop_table optional (i.e., you didn't need two tests here): As I noted earlier, the "attempt" to drop a table which doesn't succeed has absolutely no effect (and your test verifies that, partially), so you can do it as part of the regular test - you don't need to repeat the regular part of the test twice.

Reply:
removed `attempt_drop_table` parameter in v4